src/common/matmul_pd.hpp

```diff
@@ -159,6 +159,18 @@ struct matmul_pd_t : public primitive_desc_t {
         return 1 << (wei_ndims - 2);
     }
 
+    int dst_qmask_N() const {
+        const int dst_ndims = dst_md(0)->ndims;
+        assert(dst_ndims >= 2);
+        return 1 << (dst_ndims - 1);
```

Comment:
```suggestion
        return wei_qmask_N();
```
Same can be applied to matmul.cpp